### PR TITLE
DDCNL-6416: 2022 Gov Spend

### PR DESCRIPTION
**Pull request by Christian-QA**

### CODE REVIEW

File: app/models/Year.scala [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    override def writes(o: Year): JsValue = JsNumber(o.year)
+  }
+
+  implicit val reads: Reads[Year] = new Reads[Year] {
+    override def reads(json: JsValue): JsResult[Year] =
+      json match {
+        case JsNumber(value) =>
+          println("3" * 100)
+          println(json)
+          println("3" * 100)
```
Comment (BetaDraconis) [inline app/models/Year.scala:38]: printlns left in code

File: app/models/Year.scala [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+          JsSuccess(
+            Year(value.toInt)
+          )
+        case _ =>
+          println("4" * 100)
```
Comment (BetaDraconis) [inline app/models/Year.scala:43]: printlns left in code

File: app/models/Year.scala [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+          JsError("Not a year") // TODO - Change
```
Comment (BetaDraconis) [inline app/models/Year.scala:44]: A TODO

File: app/models/Year.scala [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+        stringBinder.bind(key, params).map {
+          case Right(s)      => withParameter(s)
+          case Left(message) => Left(message)
+        }
+      }
+
+      def unbind(key: String, value: Year): String = stringBinder.unbind(key, value.year)
+    }
+
+//  implicit def queryStringBindable: QueryStringBindable[Year] = new QueryStringBindable[Year] {
```
Comment (BetaDraconis) [inline app/models/Year.scala:69]: Is this supposed to still be here?

**Review by BetaDraconis — CHANGES_REQUESTED** [review]

**Review by BetaDraconis — APPROVED** [review]

**Review by pascal22p — APPROVED** [review]

**Review by pascal22p — APPROVED** [review]

**Review by pascal22p — APPROVED** [review]

**Pull request merged**